### PR TITLE
fix(rest): create new endpoint for import spdx information in admin tab

### PR DESCRIPTION
**Pull request by nikkuma7**
[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.
> * Which issue is this pull request belonging to and how is it solving it? (*Refer to issue here*)
> * Did you add or update any new dependencies that are required for your change?

Issue: #2111 

### How To Test?
1: login sw360 with admin user
2: /resource/api/licenses/import/SPDX

> How should these changes be tested by the reviewer?
> Have you implemented any additional tests?

### Checklist
Must:
- [ ] All related issues are referenced in commit messages and in PR

### CODE REVIEW

File: rest/resource-server/src/docs/asciidoc/licenses.adoc [hunk 1]
```diff
@@ -97,4 +97,15 @@ A `DELETE` request will delete all licenses info.
...
 ===== Example response
-include::{snippets}/should_document_delete_all_license_info/http-response.adoc[]
\ No newline at end of file
+include::{snippets}/should_document_delete_all_license_info/http-response.adoc[]
+
+[[import-SPDX-info]]
+==== Import SPDX information
+
+A `IMPORT` request will import SPDX info.
+
```
Comment (rudra-superrr) [inline rest/resource-server/src/docs/asciidoc/licenses.adoc:106]: Change message to --> A `POST` request will import SPDX info.

File: rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/license/LicenseController.java [hunk 2]
```diff
@@ -35,6 +36,8 @@
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.multipart.MultipartFile;
 import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
```
Comment (rudra-superrr) [inline rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/license/LicenseController.java:41]: Remove the above three imports as they are not used.

File: rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/license/Sw360LicenseService.java [hunk 3]
```diff
@@ -153,4 +153,13 @@ private LicenseService.Iface getThriftLicenseClient() throws TTransportException
...
+
+	public void importSpdxInformation(User sw360User) throws TException {
+		LicenseService.Iface sw360LicenseClient = getThriftLicenseClient();
+		if (PermissionUtils.isUserAtLeast(UserGroup.ADMIN, sw360User)) {
+            RequestSummary deleteLicenseStatus = sw360LicenseClient.importAllSpdxLicenses(sw360User);
+        } else {
+            throw new HttpMessageNotReadableException("Unable to import All Spdx license. User is not admin");
+        }
+	}
 }
```
Comment (rudra-superrr) [inline rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/license/Sw360LicenseService.java:165]: Use proper indentation for the above method.

**Comment by rudra-superrr** [comment]
Rest doc is broken.

image

File: rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/license/Sw360LicenseService.java (hunk 3 above)
Comment (nikkuma7) [reply rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/license/Sw360LicenseService.java:164]: done.

File: rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/license/LicenseController.java (hunk 2 above)
Comment (nikkuma7) [reply rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/license/LicenseController.java]: done.

File: rest/resource-server/src/docs/asciidoc/licenses.adoc (hunk 1 above)
Comment (nikkuma7) [reply rest/resource-server/src/docs/asciidoc/licenses.adoc:106]: done.

**Comment by nikkuma7** [comment]
> Rest doc is broken.
> 
> image

done.

**Comment by rudra-superrr** [comment]
code looks good.

**Comment by rudra-superrr** [comment]
Testing was successful.

**Pull request merged**